yarn.lock

```diff
@@ -0,0 +1,10752 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
We are no longer using Yarn, this file needs to be deleted, please install with npm 
there should be a package-lock.json file